docs/ops/deployment/kubernetes.md

```diff
@@ -81,6 +81,10 @@ You can then access the Flink UI via different ways:
 
         {% highlight bash %}./bin/flink run -m <public-node-ip>:<node-port> ./examples/streaming/WordCount.jar{% endhighlight %}
 
+* Create a `NodePort` service on the query state service of taskmanager:
```

Comment:
```suggestion
You can also access the queryable state of TaskManager if you create a `NodePort` service for it:
```

Reply:
by using `* ` you continue the list related to:
`You can then access the Flink UI via different ways:`

Reply:
Actually i want to continue the list. I think your suggestion also make sense. I will update it.